### ISSUE TITLE
cannot import name 'AutoTokenizer' from 'transformers'错误

### ISSUE THREAD

**Issue by zmwstu** [chatchat-space/Langchain-Chatchat#3978]
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)

cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)cannot import name 'AutoTokenizer' from 'transformers' (/home/zwm/miniconda3/envs/Langchain-Chatchat2/lib/python3.11/site-packages/transformers/__init__.py)
下面是环境变量：
accelerate                               0.24.1
addict                                   2.4.0
aiofiles                                 23.2.1
aiohttp                                  3.9.3
aioprometheus                            23.12.0
aiosignal                                1.3.1
aliyun-python-sdk-core                   2.15.1
aliyun-python-sdk-kms                    2.16.2
altair                                   5.3.0
antlr4-python3-runtime                   4.9.3
anyio                                    4.3.0
argon2-cffi                              23.1.0
argon2-cffi-bindings                     21.2.0
arxiv                                    2.1.0
asgiref                                  3.8.1
attrs                                    23.2.0
azure-core                               1.30.1
azure-storage-blob                       12.19.1
backoff                                  2.2.1
bcrypt                                   4.1.2
beautifulsoup4                           4.12.3
blinker                                  1.7.0
blis                                     0.7.11
boto3                                    1.34.88
botocore                                 1.34.88
Brotli                                   1.1.0
build                                    1.2.1
cachetools                               5.3.3
catalogue                                2.0.10
certifi                                  2024.2.2
cffi                                     1.16.0
chardet                                  5.2.0
charset-normalizer                       3.3.2
chroma-hnswlib                           0.7.3
chromadb                                 0.4.24
click                                    8.1.7
cloudpathlib                             0.16.0
coloredlogs                              15.0.1
confection                               0.1.4
contourpy                                1.2.1
crcmod                                   1.7
cryptography                             42.0.5
cycler                                   0.12.1
cymem                                    2.0.8
dashscope                                1.13.6
dataclasses-json                         0.6.4
dataclasses-json-speakeasy               0.5.11
datasets                                 2.19.0
Deprecated                               1.2.14
deprecation                              2.1.0
dill                                     0.3.8
distro                                   1.9.0
duckduckgo-search                        3.9.9
effdet                                   0.4.1
einops                                   0.7.0
elastic-transport                        8.13.0
elasticsearch                            8.13.0
embeddings                               0.0.8
emoji                                    2.10.1
environs                                 9.5.0
et-xmlfile                               1.1.0
faiss                                    1.7.4
fastapi                                  0.109.0
feedparser                               6.0.10
filelock                                 3.13.3
filetype                                 1.2.0
flash-attn                               2.5.7
flatbuffers                              24.3.25
fonttools                                4.51.0
frozenlist                               1.4.1
fschat                                   0.2.35
fsspec                                   2024.3.1
ftfy                                     6.2.0
gast                                     0.5.4
gitdb                                    4.0.11
GitPython                                3.1.43
google-auth                              2.29.0
googleapis-common-protos                 1.63.0
greenlet                                 3.0.3
grpcio                                   1.60.0
h11                                      0.14.0
h2                                       4.1.0
hpack                                    4.0.0
httpcore                                 1.0.5
httptools                                0.6.1
httpx                                    0.26.0
httpx-sse                                0.4.0
huggingface-hub                          0.22.2
humanfriendly                            10.0
hyperframe                               6.0.1
idna                                     3.6
importlib-metadata                       7.0.0
importlib_resources                      6.4.0
iniconfig                                2.0.0
iopath                                   0.1.10
isodate                                  0.6.1
jieba                                    0.42.1
Jinja2                                   3.1.3
jmespath                                 0.10.0
joblib                                   1.3.2
jq                                       1.7.0
jsonpatch                                1.33
jsonpath-python                          1.0.6
jsonpointer                              2.4
jsonschema                               4.21.1
jsonschema-specifications                2023.12.1
kiwisolver                               1.4.5
kubernetes                               29.0.0
langchain                                0.0.354
langchain-community                      0.0.20
langchain-core                           0.1.23
langchain-experimental                   0.0.47
langcodes                                3.3.0
langdetect                               1.0.9
langsmith                                0.0.87
layoutparser                             0.3.4
llama-index                              0.9.35
lxml                                     5.1.0
Markdown                                 3.6
markdown-it-py                           3.0.0
markdown2                                2.4.13
markdownify                              0.11.6
MarkupSafe                               2.1.5
marshmallow                              3.20.2
matplotlib                               3.8.4
mdurl                                    0.1.2
megatron-util                            1.3.2
metaphor-python                          0.1.23
minio                                    7.2.5
mmh3                                     4.1.0
mock                                     5.1.0
modelscope                               1.13.3
monotonic                                1.6
mpmath                                   1.3.0
msg-parser                               1.2.0
msgpack                                  1.0.8
multidict                                6.0.5
multiprocess                             0.70.16
murmurhash                               1.0.10
mypy-extensions                          1.0.0
nest-asyncio                             1.6.0
networkx                                 3.3
nh3                                      0.2.17
ninja                                    1.11.1.1
nltk                                     3.8.1
numexpr                                  2.8.6
numpy                                    1.26.4
nvidia-cublas-cu12                       12.1.3.1
nvidia-cuda-cupti-cu12                   12.1.105
nvidia-cuda-nvrtc-cu12                   12.1.105
nvidia-cuda-runtime-cu12                 12.1.105
nvidia-cudnn-cu12                        8.9.2.26
nvidia-cufft-cu12                        11.0.2.54
nvidia-curand-cu12                       10.3.2.106
nvidia-cusolver-cu12                     11.4.5.107
nvidia-cusparse-cu12                     12.1.0.106
nvidia-nccl-cu12                         2.18.1
nvidia-nvjitlink-cu12                    12.4.127
nvidia-nvtx-cu12                         12.1.105
oauthlib                                 3.2.2
olefile                                  0.47
omegaconf                                2.3.0
onnx                                     1.16.0
onnxruntime                              1.15.1
openai                                   1.9.0
opencv-python                            4.9.0.80
openpyxl                                 3.1.2
opentelemetry-api                        1.24.0
opentelemetry-exporter-otlp-proto-common 1.24.0
opentelemetry-exporter-otlp-proto-grpc   1.24.0
opentelemetry-instrumentation            0.45b0
opentelemetry-instrumentation-asgi       0.45b0
opentelemetry-instrumentation-fastapi    0.45b0
opentelemetry-proto                      1.24.0
opentelemetry-sdk                        1.24.0
opentelemetry-semantic-conventions       0.45b0
opentelemetry-util-http                  0.45b0
orjson                                   3.10.0
oss2                                     2.18.4
overrides                                7.7.0
packaging                                23.2
pandas                                   2.0.3
pathlib                                  1.0.1
pdf2image                                1.17.0
pdfminer.six                             20231228
pdfplumber                               0.11.0
pikepdf                                  8.4.1
Pillow                                   9.5.0
pillow_heif                              0.16.0
pip                                      24.0
platformdirs                             4.2.0
pluggy                                   1.4.0
portalocker                              2.8.2
posthog                                  3.5.0
preshed                                  3.0.9
prompt-toolkit                           3.0.43
protobuf                                 3.20.3
psutil                                   5.9.8
psycopg2                                 2.9.9
pulsar-client                            3.5.0
pyarrow                                  15.0.2
pyarrow-hotfix                           0.6
pyasn1                                   0.6.0
pyasn1_modules                           0.4.0
pyclipper                                1.3.0.post5
pycocotools                              2.0.7
pycparser                                2.22
pycryptodome                             3.20.0
pydantic                                 1.10.13
pydeck                                   0.8.1b0
Pygments                                 2.17.2
PyJWT                                    2.8.0
pymilvus                                 2.4.0
PyMuPDF                                  1.23.16
PyMuPDFb                                 1.23.9
pypandoc                                 1.13
pyparsing                                3.1.2
pypdf                                    4.2.0
pypdfium2                                4.28.0
PyPika                                   0.48.9
pyproject_hooks                          1.0.0
pytesseract                              0.3.10
pytest                                   7.4.3
pythainlp                                5.0.2
python-crfsuite                          0.9.10
python-dateutil                          2.8.2
python-decouple                          3.8
python-docx                              1.1.0
python-dotenv                            1.0.1
python-iso639                            2024.2.7
python-magic                             0.4.27
python-multipart                         0.0.9
python-pptx                              0.6.23
pytz                                     2024.1
pyvi                                     0.1.1
PyYAML                                   6.0.1
quantile-python                          1.1
rapidfuzz                                3.6.1
rapidocr-onnxruntime                     1.3.8
ray                                      2.10.0
referencing                              0.34.0
regex                                    2023.12.25
requests                                 2.31.0
requests-oauthlib                        2.0.0
rich                                     13.7.1
rouge                                    1.0.1
rpds-py                                  0.18.0
rsa                                      4.9
s3transfer                               0.10.1
sacremoses                               0.1.1
safetensors                              0.4.2
scikit-learn                             1.4.1.post1
scipy                                    1.13.0
sentence-transformers                    2.2.2
sentencepiece                            0.2.0
seqeval                                  1.2.2
setuptools                               69.5.1
sgmllib3k                                1.0.0
shapely                                  2.0.3
shortuuid                                1.0.13
simplejson                               3.19.2
six                                      1.16.0
sklearn-crfsuite                         0.3.6
smart-open                               6.4.0
smmap                                    5.0.1
sniffio                                  1.3.1
socksio                                  1.0.0
sortedcontainers                         2.4.0
soupsieve                                2.5
spacy                                    3.7.2
spacy-legacy                             3.0.12
spacy-loggers                            1.0.5
SQLAlchemy                               2.0.25
srsly                                    2.4.8
sse-starlette                            1.8.2
stanza                                   1.8.2
starlette                                0.35.0
streamlit                                1.30.0
streamlit-aggrid                         0.3.4.post3
streamlit-antd-components                0.3.1
streamlit-chatbox                        1.1.11
streamlit-feedback                       0.1.3
streamlit_modal                          0.1.0
streamlit-option-menu                    0.3.12
strsimpy                                 0.2.1
subword-nmt                              0.3.8
svgwrite                                 1.4.3
sympy                                    1.12
tabulate                                 0.9.0
tenacity                                 8.2.3
termcolor                                2.4.0
thinc                                    8.2.3
threadpoolctl                            3.4.0
tiktoken                                 0.5.2
timm                                     0.9.16
tokenizers                               0.15.2
toml                                     0.10.2
tomli                                    2.0.1
toolz                                    0.12.1
torch                                    2.1.2
torchaudio                               2.1.2
torchvision                              0.16.2
tornado                                  6.4
tqdm                                     4.66.1
transformers                             4.37.2
transformers-stream-generator            0.0.4
triton                                   2.1.0
typer                                    0.9.4
typing_extensions                        4.9.0
typing-inspect                           0.9.0
tzdata                                   2024.1
tzlocal                                  5.2
ujson                                    5.9.0
unstructured                             0.12.5
unstructured-client                      0.18.0
unstructured-inference                   0.7.23
unstructured.pytesseract                 0.3.12
urllib3                                  1.26.18
uvicorn                                  0.29.0
uvloop                                   0.19.0
validators                               0.28.0
vllm                                     0.2.7
wasabi                                   1.1.2
watchdog                                 3.0.0
watchfiles                               0.21.0
wavedrom                                 2.0.3.post3
wcwidth                                  0.2.13
weasel                                   0.3.4
websocket-client                         1.7.0
websockets                               12.0
wheel                                    0.43.0
wrapt                                    1.16.0
xformers                                 0.0.23.post1
xlrd                                     2.0.1
XlsxWriter                               3.2.0
xxhash                                   3.4.1
yapf                                     0.40.2
yarl                                     1.9.4
youtube-search                           2.1.2
zhconv                                   1.4.3
zipp                                     3.18.1
